components/General/MarketChange/index.tsx

```diff
@@ -4,23 +4,12 @@ import styled from 'styled-components';
 interface MCProps {
     className?: string;
     amount: number;
-    size?: 'sm' | 'lg';
-    before?: boolean; // boolean to determin which side the arrow goes on
 }
 
-const MarketChange: React.FC<MCProps> = styled(({ className, amount, size, before }: MCProps) => (
+const MarketChange: React.FC<MCProps> = styled(({ className, amount }: MCProps) => (
     <div className={className}>
-        {before ? (
-            <>
-                <div className={`${amount >= 0 ? 'arrow-up' : 'arrow-down'} ${size} ${before ? 'before' : ''}`} />
-                <p className={amount >= 0 ? 'up' : 'down'}>0%</p>
-            </>
-        ) : (
-            <>
-                <p className={amount >= 0 ? 'up' : 'down'}>0%</p>
-                <div className={`${amount >= 0 ? 'arrow-up' : 'arrow-down'} ${size}`} />
-            </>
-        )}
+        <div className={amount >= 0 ? 'up' : 'down'}>{amount}%</div>
+        <div className={`${amount >= 0 ? 'arrow-up' : 'arrow-down'}`} />
```

Comment:
The before will need to be added back at some stage since its in the designs to be with the market dropdown. If you global search <MarketChange you will be able to see where its used but its commented out